### DISCUSSION TITLE
Docs small Error, so example code doesn't work

### DISCUSSION THREAD

**Discussion by hussilarabi** [lit/lit#4577] [General]
On [this page](https://lit.dev/docs/composition/controllers/#asynchronous-tasks) the code example doesn't work, because the url is dead.
For the starwars-api it should be "https://swapi.py4e.com/api/" instead of "https://swapi.dev/api/"

As the code example is embeded from [this playground page](https://lit.dev/playground/#sample=v3-docs/controllers/names) i gues the fix should be made there .

**Comment by augustjk** [comment]
Thanks for the report! It'll be fixed here: https://github.com/lit/lit.dev/pull/1309
